styles.css

```diff
@@ -0,0 +1,71 @@
+p {
+    margin: 0;
+}
+
+.credit-card {
+    background-color: #425ecd;
+    font-size: large;
+    height: 240px;
+    width: 360px;
+    border-radius: 10px;
+    margin: 30px 0 0 20px;
+    display: flex;
+    flex-direction: column;
+}
+
+.credit-card-front > div {
+    margin-left: 40px;
```

Comment:
It would be more elegant to provide padding for `credit-card-front` instead of applying margin separately for all its children

Reply:
done